argobots.spec

```
@@ -1,12 +1,18 @@
 Name: argobots
-Version: 1.1~a1
+
+%global major 1
+%global minor 1
+%global prerelease a1
+%global tag %{major}.%{minor}%{prerelease}
+Version: %{major}.%{minor}%{?prerelease:~%{prerelease}}
+
```

Move this blank line above `Version:`.
```suggestion

Version: %{major}.%{minor}%{?prerelease:~%{prerelease}}
```

---

ok